AmericanMedicalAssociation/ama-style-guide-2
# ADF-134: Prototype doctor claim profile

**Jira Ticket**
- [ADF-134: Prototype doctor claim profile](https://palantir.atlassian.net/browse/ADF-134)

## Description

- Add Dr Finder Claim molecule to the styleguide.
- Add conditions to dr finder profile header states with dr finder claim.


## To Test

- Checkout the **ADF-134-prototype-doctor-claim-profile** branch and cd into style guide, run gulp serve to spin up the pattern lab style guide.

- Verify that **dr-finder-claim** folder and related files existed under Molecules.

- Visit **http://localhost:3000/?p=viewall-molecules-dr-finder-claim** &
           **http://localhost:3000/?p=viewall-molecules-dr-finder-profile-header** and verify Dr Finder Claim modal.

## Related Task

- Add pill-claim button to the styleguide. 

## Relevant Screenshots/GIFs
<img width="570" alt="Screen Shot 2022-10-11 at 2 41 34 PM" src="https://user-images.githubusercontent.com/54925022/195194286-d55b08af-1483-4070-9eca-254fd3a4ce65.png">


![Screen Shot 2022-10-11 at 2 29 57 PM](https://user-images.githubusercontent.com/54925022/195194293-204ffe71-f6a0-45ed-973a-72ce35142dad.png)
